### ISSUE TITLE
Relabel "I am awake" button

### ISSUE THREAD

**Issue by dneise** [fact-project/shifthelper_webinterface#30]
Proposal: 

"I am ready to park within the next 20 minutes."

**Comment by maxnoe** [comment]
Way to long for a button. I would propose a tooltip explaining what it actually does.

**Comment by dneise** [comment]
Yes a tool tip would be good, but still "Ready To Park" captures better that this button is only related to the "Is Shifter Awake Before Shutdown" check, and does do anything else, like e.g. muting all checks.

Also we were thinking, in principle Acknowledging the Alert  **and** pressing the "I am awake" button is doubling the action, For this check it would be enough to simply acknowledge the alert...

**Comment by dneise** [comment]
Also people where puzzled about how often they should press this button. I think it would be good to let the button appear "pressed" as long as the SH considers the person "awake". 

I tried this using the stuff explained here:
https://www.w3schools.com/bootstrap/bootstrap_buttons.asp

But I was not able to get it done. Especially hard is testing, since I have to "docker-compose up --build" all the time, to make sure the stuff I just wrote really ends up in the container ... this takes a lot of time.
